lib/hydration.js

```diff
@@ -6,7 +6,9 @@ const dotProp = require('dot-prop');
 const hydrateOption = (options, values) => {
   options.forEach((option) => {
     if (!('_attributes' in option)) return;
-    const isMatch = values.includes(option._attributes.value);
+    const isMatch = typeof option._attributes.value === 'string'
+      ? values.map(val => val.toLowerCase()).includes(option._attributes.value.toLowerCase())
+      : values.includes(option._attributes.value);
```

Comment:
@joeosburn I'm not sure if this is a good idea, but in `ui` if the casing of the string wasn't the same the proper select dropdown option wasn't getting selected on addresses. Can you see a significant downside in making hydration of options case insensitive?

Reply:
I can't think of any reason making them case insensitive would cause a problem. I'm good with this 👍